### PR TITLE
Proposing 11/3 - 11/9 Platformer Management Week Goals

### PR DESCRIPTION
**Pull request by arb204**
Added goals for my management week.

### CODE REVIEW

File: management/Platformer.md [hunk 1]
```diff
@@ -39,8 +39,15 @@
 1. 11/3 - 11/9
 	* Manager: Austin Brothers
 	* Goals:
-		1. GOALHERE
-		...
+		1. Main: 
+		 	- Create a bugs markdown file, to monitor bugs found with the game. Start fixing bugs as they are found.
```
Comment (nfarnan) [inline management/Platformer.md:43]: If you'd prefer a markdown fine, but I would suggest that you use the "Issues" tracker included in GitHub, as it is designed for this purpose.

File: management/Platformer.md [hunk 2]
```diff
@@ -39,8 +39,15 @@
...
-		...
+		1. Main: 
+		 	- Create a bugs markdown file, to monitor bugs found with the game. Start fixing bugs as they are found. 
+		 	- Implement a system to support multiple levels.
+		2. Networking:
+			- Have host and remote instances communicate across two separate systems.
+			- Send the rest of the important data to remote instance, such as animations, portals, objects, etc.
+		3. Physics:
+			- Add block and player's ability to carry block. Implement collision resolutions for block object.
+			- Start working on conserving forces such as momentum through portals.
```
Comment (nfarnan) [inline management/Platformer.md:50]: Need a more specific goal. What do you want to have implemented this week to demonstrate progress here?

**Review by nfarnan — CHANGES_REQUESTED** [review]

File: management/Platformer.md (hunk 2 above)
Comment (arb204) [reply management/Platformer.md:50]: Ok I'll work on fixing this now.

**Comment by arb204** [comment]
I updated the goals for this week to be more specific. Hopefully, their more up to expectations now.

**Review by nfarnan — APPROVED** [review]

**Pull request merged**